spec/validation/auth/authSpec.coffee

```diff
@@ -1,6 +1,6 @@
 auth = require('../../../lib/validation/validate').auth
 
-super_admin         = {name: 'admin'}
+system_user         = {name: 'admin'}
 team_admin          = {name: 'etkdg394hpmujn', roles: ['kratos|enabled']}
```

Comment:
couchdb requires that roles be an array. It also appears to expect each item in the array to be a string, even though that is not explicitly documented. to be on the safe side, I decided to just use the 'resource|role' convention.